packages/host/src/lib/host.ts

```diff
@@ -1011,5 +1041,38 @@ export class Host implements IComponent {
                 .close();
         });
     }
-}
 
+    async setTelemetry(): Promise<void> {
+        if (this.config.telemetry.status === "ask") {
+            await new Promise<void>((resolve, _reject) => {
+                require("readline").createInterface({
+                    input: process.stdin,
+                    output: process.stdout
+                }).question("Do You consent to the transmission of anonymous usage data to help us improve Transform Hub? (y/n) ", (reply: string) => {
+                    this.logger.info("User replied", reply);
+
+                    this.config.telemetry.status = ["yes", "y"].includes(reply.trim()) ? "on" : "off";
+                    resolve();
+                });
+            });
+        }
+
+        if (this.config.telemetry.status === "on") {
```

Comment:
Patryk, we were supposed to add a message about telemetry and on how to switch it off.

Reply:
Either there's one or Patryk added but in any case it displays nicely :) So I mark it as resolved. 
![Screenshot from 2022-09-08 11-25-38](https://user-images.githubusercontent.com/8177865/189086772-9a1473c9-f74c-4298-8587-3543bd6e5527.png)